tests/vmi_configuration_test.go

```diff
@@ -2989,6 +2989,33 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			doesntExceedMemoryUsage(&processRss, "qemu-kvm", qemuExpected)
 		})
 	})
+
+	Context("When topology spread constraints are defined for the VMI", func() {
+		It("they should be applied to the launcher pod", func() {
+			vmi := tests.NewRandomVMI()
+			tsc := []k8sv1.TopologySpreadConstraint{
+				{
+					MaxSkew:           1,
+					TopologyKey:       "zone",
```

Comment:
Should we use the available key in our testing cluster?

Reply:
@xpivarc Whether the key actually exists on the nodes is not relevant for this test.